notes/1.0.0.markdown

```diff
@@ -1,5 +1,137 @@
 See [sbt 1.0 roadmap and beta-1][sbt-1-0-roadmap] for scheduled timeline.
 
+### sbt 1.0.0-M6
+
+This is the beta-2 release of sbt 1.0.
+
+### Changes since sbt 1.0.0-M5
+
+#### Features, fixes, changes with compatibility implications
+
+We are working with Scala Center to provide [an automatic migration tool][sbt-migration-rewrites].
+
+- sbt 1.0 renames `Global` as scope component to `Zero` to disambiguate from `GlobalScope`. [@eed3si9n][@eed3si9n]
+
+#### Fixes
+
+- Fixes test content log not showing up. [#3198][3198]/[util#80][util80] by [@eed3si9n][@eed3si9n]
+- Fixes confusing log about "Unable to parse". [lm#98][lm98] by [@jvican][@jvican]
+- Fixes `console` task. [zinc#295][zinc295] by [@dwijnand][@dwijnand]
+- Fixes spurious recompilations when unrelated constructor changes. [zinc#288][zinc288] by [@smarter][@smarter]
+- Fixes restligeist macro for old operators. [#3218][3218] by [@eed3si9n][@eed3si9n]
+- Fixes task caching of `update` task. [#3233][3233] by [@eed3si9n][@eed3si9n]
+
+#### Improvements
+
+- Scala Center contributed a Java-friendly Zinc API. This was a overhaul of the Zinc internal API for a good Scala integration with other build tools. [zinc#304][zinc304] by [@jvican][@jvican]
+- Ivy engine with parallel artifact download. See below.
+- Scala Center contributed static validation of `build.sbt`. See below
+- Revamped documentation for [Scopes](www.scala-sbt.org/0.13/docs/Scopes.html), and added [Scope Delegation](www.scala-sbt.org/0.13/docs/Scope-Delegation.html). [@eed3si9n][@eed3si9n]
+- Ports sbt-cross-building's `^` and `^^` commands for plugin cross building. See below.
+- Adds support for cross-versioned exclusions. [#1518][1518]/[lm#88][lm88] by [@jvican][@jvican]
+- Adds new offline mode to the Ivy-based library management. [lm#92][lm92] by [@jvican][@jvican]
+- A number of features related to dependency locking. See below.
+- Improved eviction warning presentation. See below.
+- A better main class detection. [zinc#287][zinc287] by [@smarter][@smarter]
+- For faster startup, sbt will use Java refection to discover `autoImport` . [#3115][3115] by [@jvican][@jvican]
+- For faster startup, reuse the same global instance for parsing. [#3115][3115] by [@jvican][@jvican]
+- Adds `InteractionService` from sbt-core-next to keep compatibility with sbt 0.13. [#3182][3182] by [@eed3si9n][@eed3si9n]
+
+#### Static validation of build.sbt
+
+sbt 1.0 prohibits `.value` calls inside the bodies of if and else expressions in a task. `@sbtUnchecked` annotation can be used to override the check.
```

Comment:
Inside the bodies of if expressions in a task, and inside the body of an anonymous function for all types of tasks :smile: